### ISSUE TITLE
rendering of german umlauts ü and ß

### ISSUE THREAD

**Issue by jeffrson** [microsoft/terminal#2426]
# Environment

```none
Windows build number: 10.0.18362.0
Windows Terminal version (if applicable): 0.3.2171.0
```

# Steps to reproduce

`ping www.github.com` with german localization

# Expected behavior

just like in old cmd:
![image](https://user-images.githubusercontent.com/9551280/63011096-e8f8e980-be87-11e9-8a11-a9e09f59777f.png)

# Actual behavior

![image](https://user-images.githubusercontent.com/9551280/63011185-16de2e00-be88-11e9-9c72-c6031e83468e.png)

As it seems, you could also just type the (small) 'ü' and the 'ß' or copy it from editor and get the same margins. It looks like those letters are composed from two chars: moving the cursor through the line shows this. When I edit such line (backspace) or press escape (in powershell), some chars remain at the (former) right side of the text.

For capital 'Ü', ä, Ä, ö, Ö the behaviour appears to be correct.

**Comment by j4james** [comment]
I think you're right about this being two characters. If I use a precomposed u-with-diaeresis character then I don't get this problem. You have to be using a combining character for the diaeresis to get the weird spacing. 

In a WT PowerShell window, compare this:

    "uu$([char]0x0308)uu"

with this:

    "u$([char]0xFC)uu"

![image](https://user-images.githubusercontent.com/4181424/63039244-cd5b0680-beba-11e9-84e2-343a39c1ef00.png)

I think this is a known problem - @DHowett-MSFT talks about it in issue [#2003](https://github.com/microsoft/terminal/issues/2003#issuecomment-513420922) - although I'm not sure that particular issue is reporting the same thing.

As for the ß character, I don't see anything wrong with that, but whatever you're seeing could be a font specific problem. In that case, it might be related to issue #2066.

**Comment by jeffrson** [comment]
Thanks so far!

Both, 'ü' and 'ß', behave alike for me.

I too was thinking about font issue (though it seems strange, that it should make to chars out of one). I tried to change the font (I ususally have "Input Mono"), but the margin did not disappear. However, when I start the terminal while the font is "Consolas", the problem seems to vanish?!
(at least at my home PC now)

**Comment by DHowett-MSFT** [comment]
Yeah, if it changes based on what font is in use that'll be fixed when we think more about #2066.

**Comment by DHowett-MSFT** [comment]
Closing in favor of those issues. If it's a combining character, 2003. If not, 2066.
